### PR TITLE
Added Sep 2019 community newsletter

### LINKED ISSUE #38: September 2019 Python Pune Newsletter Collection Box
Let's use this issue to collect python related news, new package updates and events happening around Pune. We can compile and publish it during July pythonpune meetup.

## How to add news:
Comment on this issue with news link and news topic and also include the source from we got this.

## How to add new package release:
Paste the link of new package release

## How to include events here:
if a new event is happening in Pune and want to share in the newsletter, feel free to drop the event link with date and venue.

### PR DESCRIPTION
**Pull request by raukadah**
Closes #38

Signed-off-by: Chandan Kumar (raukadah) <raukadah@gmail.com>

### CODE REVIEW

**Comment by akshay196** [comment]
Thanks @raukadah :+1:

File: 2019/September/community_news.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* Sunsetting Python 2 
+  https://www.python.org/doc/sunset-python-2/
+
+* Vote to promote Joannah Nanjekye as a core dev 
+  https://discuss.python.org/t/vote-to-promote-joannah-nanjekye-as-a-core-dev/2347
+
+* Coverage.py 5.0a7, and the future of pytest-cov 
+  https://nedbatchelder.com/blog/201909/coveragepy_50a7_and_the_future_of_pytestcov.html
+
+* Introduction to Python asyncio for Hackers
```
Comment (bhavin192) [inline 2019/September/community_news.md]: Should add name of the author

File: 2019/September/community_news.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  https://github.com/girish946/local-paste
+
+## Upcoming events in & around Pune
+
+* PyCon India 2019 on 12th & 13th Oct, 2019 
+  https://in.pycon.org/2019/
+
+## Thanks
+
+* Thanks to shoptimize for providing venue for Sept pythonpune meetup 2019
```
Comment (bhavin192) [inline 2019/September/community_news.md]: We should make it 'Sep' and 'PythonPune'

Comment (chkumar246) [reply 2019/September/community_news.md]: Done

**Comment by raukadah** [comment]
Thanks @bhavin192 @akshay196 for the comments, Addressed .

**Pull request merged**